open-craft/opencraft
# [BB-3867] Add new theme footer update page and sidebar

**Description**

Implements the footer for the new console theme. The footer is moved to the sidebar. 

Current console:

![Selection_009](https://user-images.githubusercontent.com/13065283/111535560-4a76ed80-877a-11eb-8609-d73150050460.png)


New console:

![Screenshot_20210317_233203](https://user-images.githubusercontent.com/13065283/111535358-126faa80-877a-11eb-8551-bf2586921207.png)

**Supporting information**

* Jira Ticket: [BB-3867](https://tasks.opencraft.com/browse/BB-3867)
* [Description](https://gitlab.com/opencraft/dev/opencraft/-/issues/719) on Gitlab
* [Example](https://app.zeplin.io/project/5ea850e04326be273e4bcf5a/screen/5ffea861fa26d61b53abb3b9)

NB: The scope of this ticket does not include implementing the preview box. See [this](https://tasks.opencraft.com/browse/BB-3867?focusedCommentId=195004&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-195004) comment on the ticket.

**Testing instructions**

1. Log into the console.
2. Head over to http://localhost:3000/newconsole/theming/footer/sidebar (new console)
3. Open a new tab at http://localhost:3000/console/theming/footer (old console)
4. Update the colors at the new console and refresh the page on the old console to confirm that it indeed reflects.
5. Confirm the styling of the sidebar is as documented.

**Deadline**

End of sprint BB.242
